plugins/retry/src/main/java/forklift/retry/RetryHandler.java

```diff
@@ -47,8 +49,9 @@ public RetryHandler(ForkliftConnectorI connector) {
     public RetryHandler(ForkliftConnectorI connector, File dir) {
         this.dir = dir;
         this.connector = connector;
-        this.mapper = new ObjectMapper();
-        this.mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+        this.mapper = new ObjectMapper().registerModule(new JavaTimeModule())
+                                        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                                        .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
I'm on the fence on this one. The idea behind retries is that a message will be re-executed at a later time exactly the same as the first time. If the bean can't be converted in full it's possible we lose some data that would affect message processing.